docs/troubleshooting/harvester.md

```diff
@@ -4,7 +4,53 @@ sidebar_label: Harvester
 title: ""
 ---
 
-## Generate a support bundle
+## Fail to Deploy a Multi-node Cluster due to Incorrect HTTP Proxy Setting
+
+### ISO Installation Without a Harvester Configuration File
+
+#### Configure HTTP Proxy During Harvester Installation
+
+In some environments, you configure [http-proxy](../airgap/#configure-an-http-proxy-during-installation) of [OS Environment](../install/harvester-configuration.md#osenvironment) during Harvester installation.
+
+#### Configure HTTP Proxy After First Node is Ready
+
+After the first node is installed successfully, you login into the `Harvester GUI` to configure [http-proxy](../airgap.md#configure-an-http-proxy-in-harvester-settings) of [Harvester System Settings](../install/harvester-configuration.md#system_settings).
+
+Then you continue to add more nodes into the cluster.
+
+#### One Node Becomes Unavailable
+
+The issue you may encounter:
+
+```
+The first node is installed successfully.
+
+The second node is installed successfully.
+
+The third node is installed successfully.
+
+Then the second node changes to Unavialable state and cannot recover automatically.
+```
+
+#### Solution
+
+When the nodes in the cluster do not use the HTTP Proxy to communicate with each other, after the first node is installed successfully, you need to configure [http-proxy.noProxy](../airgap.md#configure-an-http-proxy-in-harvester-settings) against the CIDR used by those nodes.
+
+For example, your cluster assigns IPs from CIDR `172.26.50.128/27` to nodes via DHCP/static setting, please add this CIDR to `noProxy`.
+
+After this setting, you can continue to add new nodes to the cluster.
+
+For more details, please refer to [Harvester issue 3091](https://github.com/harvester/harvester/issues/3091).
+
+### ISO Installation With a Harvester Configuration File
+
+When a Harvester configuration file is used in ISO installation, please configure proper `http-proxy` in [Harvester System Settings](../install/harvester-configuration.md#system_settings).
```

Comment:
```suggestion
When a Harvester configuration file is used in an ISO installation, please configure `http-proxy` in [Harvester System Settings](../install/harvester-configuration.md#system_settings).
```

It should be implied that any configuration made should be proper/correct, so it's safe to drop `proper` here.

Reply:
It is also not necessary to add `an` to `ISO installation` in this context.  

![image](https://user-images.githubusercontent.com/31133476/202416066-2786bdc5-953d-4459-aa79-668cb9a73136.png)


The `proper` also refers to what are emphasised here, prefer to keep it.